spyder/plugins/editor/utils/decoration.py

```diff
@@ -58,7 +58,7 @@ def add(self, decorations):
             self.update()
         return added
 
-    def remove(self, decoration):
+    def remove(self, decoration, update=True):
         """
```

Comment:
@impact27 please add an explanation (as long and complete as necessary) of what update does, and how it is used.

Reply:
Done